mainchain-follower/db-sync-follower/src/db_model.rs

```diff
@@ -527,3 +527,36 @@ pub(crate) async fn index_exists(pool: &Pool<Postgres>, index_name: &str) -> boo
 		.map(|rows| rows.len() == 1)
 		.unwrap()
 }
+
+#[derive(Debug, Clone, sqlx::FromRow, PartialEq)]
+pub(crate) struct ReleasedTokens(pub Option<StakeDelegation>);
+
+#[cfg(feature = "native-token")]
+pub(crate) async fn get_native_token_releases(
```

Comment:
It doesn't return "releases" but "sum of release" tokens. Anyway I think that db query is far from business logic and the name should reflect what it does. At Cardano level there are no "releases".

This query sums value of specific tokens transfered to specific address.

Reply:
renamed